.storybook/stories.css

```diff
@@ -1,4 +1,5 @@
 @tailwind base;
+@import "../assets/index.css";

```

Comment:
Shouldn't the custom CSS come after all tailwind?

Reply:
no, according to the docs this is where they're supposed to be and it's in line with what we have in listings. dind't re-check, speaking from memory